Source/WebCore/rendering/svg/RenderSVGResourceMarker.cpp

```diff
@@ -94,21 +96,16 @@ FloatPoint RenderSVGResourceMarker::referencePoint() const
 
 float RenderSVGResourceMarker::angle() const
 {
-    float angle = -1;
-    if (markerElement().orientType() == SVGMarkerOrientAngle)
-        angle = markerElement().orientAngle().value();
-
-    return angle;
+    return markerElement().orientAngle().value();
 }
```

Comment:
I would suggest making this function return `std::optional<float>` since the `markerElement()` is not necessarily has an angle.
```
std::optional<float> RenderSVGResourceMarker::angle() const
{
    if (markerElement().orientType() == SVGMarkerOrientAngle)
        return markerElement().orientAngle().value();
    return set::null opt;
}
```

Reply:
Done and added stuff in `.h` accordingly as well.